algolia-crawler.js

```diff
@@ -0,0 +1,117 @@
+// This is a hardcopy of the Algolia crawler configuration for reference
+// The live version is authored here:
+// https://crawler.algolia.com/admin/crawlers/1b5189dd-f93c-43b7-8b45-e5ec6b0d319d/configuration/edit
+/* eslint-disable */
+
+new Crawler({
```

Comment:
So this would be used in a follow up?

Reply:
This is the server side code running on the algolia crawlers, this file is here for documentation and properly tracking changes we do.  If we accidentally delete the algolia instance we'd have to recreate this from scratch if it's not documented. 

To make clearer that it's actually docs and not code we could move it into a `/docs` folder or make it a big code block in a markdown file.  The "IDE" in the algolia backend is better for editing anyways.